wandb/apis/public/artifacts.py

```diff
@@ -1009,6 +1056,10 @@ def more(self):
     @property
     def cursor(self):
         if self.last_response:
+            if self.query_via_membership:
+                return self.last_response["project"]["artifactCollection"][
```

Comment:
this line is not covered, consider if this is important to test.

Reply:
Augmented the existing test to ensure this codepath works correctly, thanks for pointing it out!